tests/jwksClient.tests.js

```diff
@@ -16,15 +16,15 @@ describe('JwksClient', () => {
     it('should handle errors', done => {
       nock(jwksHost)
         .get('/.well-known/jwks.json')
-        .reply(500, 'Unknown Server Error');
+        .reply(500, 'Request failed with status code 500');
 
       const client = new JwksClient({
         jwksUri: `${jwksHost}/.well-known/jwks.json`
       });
 
       client.getKeys(err => {
         expect(err).not.to.be.null;
-        expect(err.message).to.equal('Unknown Server Error');
+        expect(err.message).to.equal('Request failed with status code 500');
```

Comment:
You could add an assertion that error is of type `JwksError`

Reply:
Adding the `JwksError` expectation, correctly exposes this issue https://github.com/auth0/node-jwks-rsa/pull/134#discussion_r394909843